test/resources/layout.css

```diff
@@ -152,6 +152,7 @@ body {
   width: 100%;
 }
 
+
```

Comment:
Can you revert this file since it doesn't contain changes?

Reply:
reverted.